pantheon-files-daemon/CMakeLists.txt

```diff
@@ -34,9 +35,13 @@ PACKAGES
     gio-2.0
     gee-0.8
     sqlite3
+    pantheon-files-core
+    pantheon-files-core-C
 OPTIONS
+    --vapidir=${CMAKE_SOURCE_DIR}/libcore/
```

Comment:
This line is not required for successful compilation, although it does not harm.